Core/Utilities/MySQL.py

```diff
@@ -458,7 +458,7 @@ def __escapeString( self, myString ):
     It also includes quotation marks " around the given string
     """
 
-    retDict = self.__getConnection()
+    retDict = self._getConnection()
```

Comment:
Yes. It is simplified but it is complicated the debug. We know all the services are using _getConnection() method. MySQL is using __getConnection internally. So If we want to debug the mysql db, we can not do it easily as now we have calls from services..